arm/.global/global.module.tests.ps1

```diff
@@ -49,8 +49,8 @@ Describe 'File/folder tests' -Tag Modules {
 
                 $workflowsFolderName = Join-Path $RepoRoot '.github' 'workflows'
                 $workflowFileName = '{0}.yml' -f $moduleFolderName.Replace('\', '/').Replace('/', '.').Replace('Microsoft', 'ms').ToLower()
-
-                Test-Path (Join-Path $workflowsFolderName $workflowFileName) | Should -Be $true
+                $workflowPath = Join-Path $workflowsFolderName $workflowFileName
```

Comment:
Is this related to this PR?

Reply:
Yes. The teast wasn't as meaningful as it should have been